lib/puppet/provider/java_ks/keytool.rb

```diff
@@ -93,7 +93,7 @@ def import_ks
       '-srckeystore', tmppk12.path,
       '-alias', @resource[:name]
     ]
-    cmd << '-trustcacerts' if @resource[:trustcacerts] == :true
+    cmd << '-trustcacerts' if @resource[:trustcacerts]
```

Comment:
Nope.  Unsafe rubocop change was unsafe.  This is now broken.  If in a manifest, the parameter is explicitly set to `false`, `@resource[:trustcacerts]` will still be the symbol `:false` here and be evaluated as truthy.